eng/bomgenerator/samplereport.txt

```diff
@@ -0,0 +1,21 @@
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - We dropped the following dependencies from the input list.
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-chat:1.0.0, Reason com.azure:azure-core:1.14.1
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-common:1.0.3, Reason com.azure:azure-core-http-netty:1.10.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-identity:1.1.1, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-phonenumbers:1.0.3, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-sms:1.0.3, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-cosmos:4.17.0, Reason org.slf4j:slf4j-api:1.7.31
```

Comment:
@alzimmermsft Cosmos is getting dropped since it uses a higher version of slf4j than core. I believe this is a conflct in our current BOM too -  Did we consciously allow this because it is only a logging API?

Reply:
This may have been due to version updates happening between releases

Reply:
Thanks.